kafka_consumer/tests/docker/kerberos/docker-compose.yaml

```
@@ -0,0 +1,132 @@
+version: '3.9'
+
+services:
+  kdc:
+    hostname: kdc.kerberos-demo.local
+    build: ./kdc
+    container_name: kdc
+    ports:
+      - "88:88"
+      - "749:749"
+    volumes:
+      - ${SECRET_DIR}:/var/lib/secret
+      - ./kdc/krb5.conf:/etc/kdc/krb5.conf
+    healthcheck:
+      test: bash -c "[ -f /var/lib/secret/localhost.key ]"
+      start_period: 5s
+      interval: 5s
+      timeout: 5s
+      retries: 10
+
+  zookeeper:
+    hostname: zookeeper.kerberos-demo.local
+    container_name: zookeeper
+    image: confluentinc/cp-zookeeper:7.3.0
+    depends_on:
+      kdc:
+        condition: service_healthy
+    healthcheck:
+      test: nc -vz localhost 2181 || exit -1
+      start_period: 5s
+      interval: 5s
+      timeout: 5s
+      retries: 10
+    volumes:
+      - ${SECRET_DIR}:/var/lib/secret
+      - ./kdc/krb5.conf:/etc/krb5.conf
+      - ./zookeeper/zookeeper.sasl.jaas.config:/etc/kafka/zookeeper.sasl.jaas.config
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      KAFKA_OPTS: -Djava.security.auth.login.config=/etc/kafka/zookeeper.sasl.jaas.config
```

This will make it so that every new line counts as a single space and will exclude the last:

```suggestion
      KAFKA_OPTS: >-
        -Djava.security.auth.login.config=/etc/kafka/zookeeper.sasl.jaas.config
```